com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/ResolveMainClassHandler.java

```diff
@@ -141,6 +144,15 @@ public void acceptSearchMatch(SearchMatch match) {
         return resolutions;
     }
 
+    private boolean isContainedInInvisibleProject(IProject project, Collection<IPath> rootPaths) {
+        if (project == null) {
+            return false;
+        }
+
+        IFolder workspaceLinkFolder = project.getFolder("_");
```

Comment:
Use `org.eclipse.jdt.ls.core.internal.ProjectUtils.WORKSPACE_LINK` to replace `"_"`

Reply:
resolved